### PR TITLE
pin go-bindata at 3.1.2 and vendoring it

### PR DESCRIPTION
**Pull request by yjst2012**

### CODE REVIEW

File: cmd/gencosmosdb/gencosmosdb.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"unicode"
 
+	_ "github.com/go-bindata/go-bindata"
```
Comment (m1kola) [inline cmd/gencosmosdb/gencosmosdb.go:11]: This is not being used directly as a library in `go-cosmosdb`. It is just a dev tool. So it should not be imported here. I think we should use the same pattern as in ARO-RP (with `deps.go`).

**Review by m1kola — CHANGES_REQUESTED** [review]
This is what we need, but IMO we should do it in the same way we do it in ARO-RP.

File: cmd/gencosmosdb/gencosmosdb.go (hunk 1 above)
Comment (yjst2012) [reply cmd/gencosmosdb/gencosmosdb.go]: Code updated, thanks Mikalai.

**Comment by yjst2012** [comment]
given this is a quite straightforward change I just make it ready for review, hopefully not many issues/problems left.

File: pkg/gencosmosdb/generate.go [hunk 2]
```diff
@@ -1,3 +1,3 @@
 package gencosmosdb
 
-//go:generate go-bindata -pkg gencosmosdb -prefix cosmosdb -nometadata cosmosdb
+//go:generate go run ../../vendor/github.com/go-bindata/go-bindata/go-bindata -nometadata -pkg gencosmosdb -prefix cosmosdb -nometadata cosmosdb
```
Comment (m1kola) [inline pkg/gencosmosdb/generate.go:3]: `-nometadata` is present twice. Is this intentional?

Comment (yjst2012) [reply pkg/gencosmosdb/generate.go]: good catch, fixed it just now, thank you!

**Review by m1kola — APPROVED** [review]
@yjst2012 looks good to me. Please update ARO-RP to use the latest version. Thanks

**Pull request merged**